man/fi_psm.7.md

```diff
@@ -21,6 +21,18 @@ tag-matching message queue functions and the Active Message functions
 to support a variety of libfabric data transfer APIs, including tagged
 message queue, message queue, RMA, and atomic operations.
 
+The *psm2* provider runs over the PSM 2.0 interface that is
```

Comment:
You might want to add a `man/man7/fi_psm2.7` file that just contains

```man
.so man7/fi_psm.7
```

That way, people can do `man fi_psm` or `man fi_psm2`, and they'll see the same man page.